content/800-data-platform/050-data-proxy/700-prisma-cli-with-data-proxy.mdx

```diff
@@ -35,29 +25,42 @@ If your database is behind a firewall, add your local IP address to the allowlis
 - `prisma db execute`
 - `prisma studio`
 
-### Solution
+In Prisma versions 4.9.0 and later, you can [set a direct database connection URL in your Prisma schema](#set-a-direct-database-connection-url-in-your-prisma-schema).
 
-Assuming that you have `env("DATABASE_URL")` in your `schema.prisma` file, you must override the `DATABASE_URL` environment variable before you run `npx prisma migrate deploy` or any of the commands in the list above.
+In earlier versions, you must add an [environment variable](/guides/development-environment/environment-variables) with the direct database connection URL and overwrite the [`DATABASE_URL` environment variable](/guides/development-environment/environment-variables#example-set-the-database_url-environment-variable-in-an-env-file) when you need a direct connection.
 
-You can declare the environment variable `MIGRATE_DATABASE_URL`, and use that to override the connection.
+<Admonition type="warning">
 
-For example:
+**Important**<br /><br />
 
-```bash file=.env highlight=2;add
-DATABASE_URL="prisma://..."
-MIGRATE_DATABASE_URL="postgresql://..."
+If your database is behind a firewall, add your local IP address to the allowlist of your database.
 
-```
+</Admonition>
+
+</TopBlock>
+
+## Set a direct database connection URL in your Prisma schema
```

Comment:
```suggestion
## Set a direct database connection URL in Prisma schema
```

Reply:
"your Prisma schema" sounds better to me and in line with other docs... this is about your specific schema, rather the product in general